File: My_All_Programs/Python/fibbanocci.py
##n=int(input("Enter a number: "))
##first=0
##second=1
##for i in range(0,n):
##    print(first,end=" ")
##    temp=first
##    first=second
##    second=temp+second


n=int(input("Enter a number: "))
first=0
second=1
l=[]
for i in range(0,n):
    l.append(first)
    temp=first
    first=second
    second=temp+second
if(n in l):
    print("True")
else:
    print("False")

File: My_All_Programs/Java/arrays/Fibbanocci.java
package arrays;
import  java.util.Scanner;
public class Fibbanocci 
{

	public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);
		int first=0,second=1,num,i,j,temp;
		System.out.print("Enter the number: ");
		num = sc.nextInt();
		for(i=0;i<num;i++)
		{
			System.out.print(first+" ");
			temp=first +second;
			first=second;
			second=temp;
		}
		
	}

}
